<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddCommodities</class>
 <widget class="QDialog" name="FormAddCommodities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtID">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAmount">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="amount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPrice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateExp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dateexp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Exp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
